defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# defaults file for sanity-check
+
+SANITY_CHECK_DISK_SPACE:
+    # Mount point
+  - mount: /
```

Comment:
Could there be a '*' value which would check all mounted drives? That would be a useful default here, to not have to remember updating this every time a partition changes or is added.